.travis.yml

```diff
@@ -89,32 +92,33 @@ install:
   # I will remove the sphinx downgrade. Change setup.py requirements when it
   # be the case, Katta
   - echo "------- Install Dependencies -------"
-  - pip3 install -U pip setuptools
-  - pip3 install Jinja2 PyYAML pytest sympy --user
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
-      pip3 install -U --user sphinx==2.4.4 nbsphinx>=0.3.2 m2r sphinx-rtd-theme jupyter;
-    fi
+  - pip install -U pip setuptools scikit-build Jinja2 PyYAML pytest "sympy<1.6"
 
 #=============================================================================
 # Build, test and install
 #=============================================================================
 script:
   - echo "------- Build, Test and Install -------"
-  - mkdir build && cd build
-  - cmake .. -DPYTHON_EXECUTABLE=$(which python3) -DCMAKE_INSTALL_PREFIX=$HOME/nmodl
-  - make -j 2
-  - make test
+  - mkdir build && pushd build
+  - PYTHONPATH=$(python -c "import sys; print(':'.join(sys.path))") cmake .. -DPYTHON_EXECUTABLE=$(which python) -DCMAKE_INSTALL_PREFIX=$HOME/nmodl
+  - make -j 2;
+    if [ $? -ne 0 ]; then
+        make VERBOSE=1;
+    fi
+  - env CTEST_OUTPUT_ON_FAILURE=1 make test
   - make install
+  - popd
 
 #=============================================================================
 # Build Documentation, CoreNEURON and run tests
 #=============================================================================
 after_success:
-  - export PYTHONPATH=$HOME/nmodl/lib/python:$PYTHONPATH
+  # Scikit-build does not support build_ext --inplace in custom commands.
+  # More info at: https://github.com/scikit-build/scikit-build/issues/489
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
       echo "------- Build Documentation -------";
+      python setup.py build_ext --inplace docs -j 2 -G "Unix Makefiles";
```

Comment:
nitpick: after executing `python3.7 setup.py build_ext --inplace docs -j 2 -G "Unix Makefiles";` I see:

```
...
-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    SKBUILD
```
Is this something we specify?

Reply:
No. It is used to test if we are running using SKBUILD. From the [documentation](https://scikit-build.readthedocs.io/en/latest/usage.html#basic-usage):

> How to test if scikit-build is driving the compilation ?
To support the case of code base being built as both a standalone project and a python wheel, it is possible to test for the variable SKBUILD:

> ```
> if(SKBUILD)
>   message(STATUS "The project is built using scikit-build")
> endif()
> ```